src/_blog/rust-libp2p-052-release.md

```diff
@@ -0,0 +1,233 @@
+---
+tags:
+- libp2p
+- rust
+title: rust-libp2p v0.52 release spotlight
+description: Walking through the rust-libp2p v0.52 release
+date: 2023-06-19
+permalink: "/2023-06-19-rust-libp2p-052"
+translationKey: ''
+header_image: /rust-libp2p-2022-header.png
+author: Thomas Eizinger
+---
+
+# rust-libp2p v0.52 release spotlight
+
+With the `v0.52` release, the `rust-libp2p` maintainer team tried something new.
```

Comment:
I find this paragraph difficult to read. "tried something new. Breaking changes" sounds like we re-invented the idea of breaking changes.

The below makes it a bit more explicit. Though I would advocate for moving the entire paragraph towards the end of the section.

```suggestion
With the `v0.52` release, the `rust-libp2p` maintainer team tried something new - holding back breaking changes, releasing as much as possible in a non-breaking way.
```

Reply:
Yeah I am not particularly happy with the introduction actually. I want to make these posts a recurring thing so maybe it is better to just dive right in and talk about the changes?

Reply:
Diving right in sounds good to me.